content/helpers.ts

```diff
@@ -384,6 +393,58 @@ export function getCombinationBetweenTwoItems(sourceItem: ItemConfig, targetItem
   return result;
 }
 
+// recipe functions
+export function getAllRecipesForInteraction(interaction: Interaction): Recipe[] {
+  const recipes: Recipe[] = [];
+
+  for (const recipe of allRecipes) {
+    if (recipe.interaction === interaction)
+      recipes.push(recipe as Recipe);
```

Comment:
this could be `recipes = allRecipes.filter(r => recipe.interaction === interaction)`